vuepress/docs/next/docs/curate/ecr-bundle-details.md

```diff
@@ -123,11 +123,18 @@ Here is an example of a plugin descriptor:
         role: manage-users
       - clientId: realm-management
         role: view-users
+    environmentVariables: # a set of environment variables that can be set inline or reference secrets in the namespace
+      - name: MY_SIMPLE_VAR
+        value: mySimpleValue
```

Comment:
Do we want to add an example referencing a secret? [Here](https://github.com/firegloves-bundles/entando-qe-bundle-01/blob/0.0.7/plugins/entandoqebundle01-plugin.yaml) you can find an example